data/net/minecraft/world/entity/ai/memory/WalkTarget.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/world/entity/ai/memory/WalkTarget
+	METHOD <init> (Lnet/minecraft/core/BlockPos;FI)V
+		COMMENT Constructs a walk target that tracks a position
+		ARG 1 pos
+		ARG 2 speedModifier
+		ARG 3 closeEnoughDist
+	METHOD <init> (Lnet/minecraft/world/entity/Entity;FI)V
+		COMMENT Constructs a walk target that tracks an entity's position
```

Comment:
These constructor Javadocs seem a little redundant, this information could be a doc for the class instead.